src/main/java/org/jabref/gui/JabRefGUI.java

```diff
@@ -93,7 +93,6 @@ public void start(Stage stage) {
                 dialogService.showErrorDialogAndWait("Uncaught exception occurred in " + thread, exception);
             });
         });
-
```

Comment:
I think, this is not consistent with the other style in this method, where „everything“ is separated with an empty line. Thus, can you revert this please? 😅